flake.nix

```diff
@@ -291,6 +302,28 @@
           };
         };
 
+        pkgCross = { name ? null, dirs, port ? 8000, target, craneLib }: rec {
```

Comment:
What's the port argument used for here?

Reply:
Oh. Nothing anymore. It was used in the code I copied it from for container configuration, but now it's needless. Will fix it at some point, not a big deal.